dotCMS/src/main/java/com/dotmarketing/portlets/folders/model/Folder.java

```diff
@@ -344,6 +352,26 @@ public boolean equals(Object o){
 		return true;
 	}
 
+	public ManifestInfo getManifestInfo(){
+		Folder parent = null;
+
+		try {
+			final Permissionable parentPermissionable = this.getParentPermissionable();
 
+			if (Folder.class.isInstance(parentPermissionable)) {
+				parent = (Folder) parentPermissionable;
+			}
+		} catch (DotDataException e) {
+

```

Comment:
at here a parent = null and a log

Reply:
done https://github.com/dotCMS/core/pull/20632/commits/0bdc4f20ee20291a76ab20b14f674e37a4dc4fa6